.github/workflows/delete-untagged-images.yml

```
@@ -0,0 +1,51 @@
+name: Delete Untagged GHCR Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: 'Dry run (does not delete images)'
+        required: true
+        default: 'true'
+        type: choice
+        options:
+        - 'true'
+        - 'false'
+
+env:
+  REGISTRY: ghcr.io
+  OWNER: security-tools-alliance
+  PROJECT: rengine-ng
+
+jobs:
+  delete-untagged-ghcr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ vars.GHCR_USERNAME }}
+          password: ${{ secrets.GHCR_PAT }}
+
+      - name: Delete untagged images
+        uses: Chizkiyahu/delete-untagged-ghcr-action@v3
```

```suggestion
        uses: Chizkiyahu/delete-untagged-ghcr-action@v4
```
Use latest version: https://github.com/marketplace/actions/delete-untagged-ghcr

---

upgraded